distogram/__init__.py

```diff
@@ -326,31 +334,81 @@ def stddev(h: Distogram) -> float:
     return math.sqrt(variance(h))
 
 
-def histogram(h: Distogram, bin_count: int = 100) -> List[Tuple[float, float]]:
+def histogram(
+    h: Distogram, 
+    bin_count: int = 100, 
+    data_type: str = "distogram") -> Union[List[Tuple[float, float]],
+                                           Tuple[List[float], List[float]]]:
     """ Returns a histogram of the distribution
 
     Args:
         h: A Distogram object.
         bin_count: [Optional] The number of bins in the histogram.
+        data_type: [Optional] The format of the histogram. 
+                   ("distogram" = List[Tuple[bin, count]],
+                    "numpy" = Tuple[List[counts], List[bin_edges]])
 
     Returns:
         An estimation of the histogram of the distribution, or None
         if there is not enough items in the distribution.
     """
-
     if len(h.bins) < bin_count:
         return None
 
-    bin_bounds = _linspace(h.min, h.max, num=bin_count+2)
-    counts = [count_at(h, e) for e in bin_bounds[1:-1]]
-    u = [
-        (b, new - last)
-        for b, new, last in zip(bin_bounds[1:], counts[1:], counts[:-1])
-    ]
-
+    allowed_data_types = set(["distogram", "numpy"])
+    if not data_type in allowed_data_types:
+        raise ValueError(
+            f"data_type {data_type} is not a member of {allowed_data_types}")
+
+    bin_bounds = _linspace(h.min, h.max, num=bin_count + 2)
```

Comment:
The +2 should be removed to output bin_count+1 bound values. moving to NumPy's format allows removing this inconsistency with the current output.